src/java/com/unifina/signalpath/ModuleException.java

```diff
@@ -17,28 +16,7 @@ public ModuleException(String message, Throwable cause, List<ModuleExceptionMess
 		return moduleExceptions;
 	}
 
-	public static class CompileError extends HashMap<String, Object> {
-		public CompileError(Long line, String message) {
-			put("line", line);
-			put("message", message);
-		}
+	public Map<String, Object> toMap() {
+		return new HashMap<>();
```

Comment:
Default implementation should be to render a list of `ModuleException#toMap()` results, as you are now doing in `AbstractJavaCodeWrapperModuleException`